test/e2e/openshift.cnv.workflow.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -xe
+
+# Configure kubeconfig
+export KUBECONFIG=${KUBECONFIG:-$HOME/oc4/working/auth/kubeconfig}
+export KUBECTL=${KUBECTL:-oc}
+export NAMESPACE=${NAMESPACE:-openshift-cnv}
+export SSH=${SSH:-./ssh.sh}
+export PRIMARY_NIC=${PRIMARY_NIC:-ens3}
+export FIRST_SECONDARY_NIC=${FIRST_SECONDARY_NIC:-ens7}
+export SECOND_SECONDARY_NIC=${SECOND_SECONDARY_NIC:-ens8}
+export TIMEOUT=${TIMEOUT:-60m}
+
+if [ ! -f  $SSH ]; then
+    cat << EOF > ${SSH}
+#!/bin/bash
+node_name=\${1}
+node_ip=\$($KUBECTL get node \${node_name} --no-headers -o wide | awk '{print \$6}')
+ssh core@\${node_ip} -- \${@:3}
+EOF
+    chmod +x ${SSH}
+fi
+
+# Run workflow tests
+FOCUS_1='Nodes.*when.*are.*up.*and.*new.*interface.*is.*configured.*should.*update.*node.*network.*state.*with.*it'
```

Comment:
We should backport the polarion integration and replace this by test_id selectors.

Reply:
I am into it with https://github.com/nmstate/kubernetes-nmstate/pull/453 but we have some missing commits from master that stabilize e2e tests https://github.com/nmstate/kubernetes-nmstate/pull/472